examples/MQ/serialization/data/MyHit.h

```diff
@@ -33,20 +33,25 @@ class TVector3;
 class MyHit : public FairHit
 {
   public:
-    /** Default constructor **/
-    MyHit();
+    MyHit() : FairHit() {}
```

Comment:
I think, this should work?

```suggestion
    MyHit() = default;
```

Reply:
Done.